foladipo/docs-cabinet-cp2
# Create signup and login page

#### What does this PR do?
Creates a home page where a user can sign up or login. Some, other things this PR does are:
- configure this project to be ready for Heroku deployment (#147325147).
- set up and configure Swagger (#147433725).

#### Description of task to be completed
Create a sign up or login page.

#### How should this be manually tested?
*Required*: You need to have a Postgres database server running on your computer. Then, create a `.env` according to [this template](https://github.com/andela-foladipo/docs-cabinet-cp2/blob/feature/146115889/create-endpoints-using-tdd/.env.sample).

In a terminal/command prompt on your computer, switch to the `chore/147025811/setup-frontend-dev-environment` branch of this repo. Then:
- Run each of these commands:
```bash
npm install

npm run db:migrate

npm run db:seed
```

- Then, run `npm start`.

- Note the `PORT` you set in your `.env` file for use below. If you didn't set it, its default value is `5000`.

- Open `localhost:PORT` on your machine. You should see a home page where you can sign up or log in.

#### Any background context you want to provide?
N/A

#### What are the relevant Pivotal Tracker stories?
147026181, 147325147, 147433725

#### Screenshots
N/A
